.github/workflows/python-release.yml

```diff
@@ -145,7 +145,7 @@ jobs:
           # set __version__.py files in src directory
           VERSION=$(poetry version -s)
           VERSION_FILE_CONTENT="version = '$VERSION'"
-          VERSION_FILES=$(find src -name __version__.py)
+          VERSION_FILES=$(find projects/fal/src -name __version__.py)
```

Comment:
revert this one, notice that this step has `working-directory: ${{ env.PACKAGE_DIR }}`